components/TextFields/src/MDCMultilineTextField.h

```diff
@@ -29,6 +30,9 @@
 /** A mirror of the same property that already exists on UITextField, UITextView, and UILabel. */
 @property(nonatomic, assign) BOOL adjustsFontForContentSizeCategory;
 
+/** An optional delegate for useful methods not included in UITextViewDelegate.*/
+@property(nonatomic, nullable, weak) IBOutlet id<MDCMultilineTextInputDelegate> multilineDelegate;
```

Comment:
Follow-up PR: Unit tests for encoding/decoding?

Reply:
I couldn’t figure out how. I don’t have any classes that conform.

Reply:
You could create a custom test class that conforms and encode/decode. If you wanted to be super-sneaky, slap those both into an NSArray, encode/decode the array and verify that the relationship still holds.

Reply:
Do I need to make that class in another file or can I do it in the unit test file at the top?

Reply:
I put them at the top of the test file until I have more than one. Then it's a test Supplemental?